pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -932,15 +932,20 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListeners(node *model.
 		if noneMode {
 			// do not care what the listener's capture mode setting is. The proxy does not use iptables
 			bindToPort = true
-		} else if egressListener.IstioListener != nil &&
-			// proxy uses iptables redirect or tproxy. IF mode is not set
-			// for older proxies, it defaults to iptables redirect.  If the
-			// listener's capture mode specifies NONE, then the proxy wants
-			// this listener alone to be on a physical port. If the
-			// listener's capture mode is default, then its same as
-			// iptables i.e. bindToPort is false.
-			egressListener.IstioListener.CaptureMode == networking.CaptureMode_NONE {
-			bindToPort = true
+		} else if egressListener.IstioListener != nil {
+			if egressListener.IstioListener.CaptureMode == networking.CaptureMode_NONE {
+				// proxy uses iptables redirect or tproxy. IF mode is not set
+				// for older proxies, it defaults to iptables redirect.  If the
+				// listener's capture mode specifies NONE, then the proxy wants
+				// this listener alone to be on a physical port. If the
+				// listener's capture mode is default, then its same as
+				// iptables i.e. bindToPort is false.
+				bindToPort = true
+			} else if strings.HasPrefix(egressListener.IstioListener.Bind, model.UnixAddressPrefix) {
```

Comment:
I think you want `if egressListener.IstioListener.CaptureMode == networking.CaptureMode_NONE || strings.HasPrefix(egressListener.IstioListener.Bind, model.UnixAddressPrefix)`, right?

Reply:
Yes but since the comments are big, I thought I would split them up into two separate else-ifs